node/src/components/chainspec_loader.rs

```diff
@@ -95,7 +95,7 @@ impl From<ChainspecLoader> for ChainspecInfo {
 }
 
 #[derive(Clone, DataSize, Debug, Serialize, Deserialize)]
-pub(crate) struct ChainspecLoader {
+pub struct ChainspecLoader {
```

Comment:
Is this really required?

Reply:
AFAIK yes, because we are explicitly using types from the trait `impl`s of chainspec loader. Since we the reactor is public, I believe it needs to be this way as well. I revert it to `pub(crate)`, but as soon as we use a reactor, we might have to change it.

Reply:
Removing it for now.

Reply:
Actually, scratch that. It just doesn't work without it.